images/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# update ca certificates in case they are injected with a volume mount
+function updateCerts() {
+    if [[ ! -z "$(which update-ca-certificates)" ]]; then
+        echo "Running sudo update-ca-certificates"
+        sudo update-ca-certificates
+    elif [[ ! -z "$(which update-ca-trust)" ]]; then
+        echo "Running sudo update-ca-trust"
+        sudo update-ca-trust
+    fi
+}
+
+if [[ ! -z $RUNNER_UPDATE_CA ]]; then
```

Comment:
nit: `$RUNNER_UPDATE_TRUST_STORE` might be more descriptive than `$RUNNER_UPDATE_CA`

Reply:
Updated :relaxed: :+1: